tests/unit/rate_limiting/test_core.py

```diff
@@ -119,7 +119,7 @@ def test_resets_in_expired(self, metrics):
             ]
         )
 
-        limiter._window = pretend.stub(get_window_stats=lambda l, *a: next(stats))
+        limiter._window = pretend.stub(get_window_stats=lambda L, *a: next(stats))
```

Comment:
@miketheman What bump introduced this? I'm curious why this is happening only on a few lines when we probably have lots of other instances of this.

Reply:
flake8 went from 4.0.0 to 6.0.0, guessing its that.

Reply:
Yep - links to issue in [`3c78893` (#13075)](https://github.com/pypi/warehouse/pull/13075/commits/3c7889309c189555dc221e54b7a2d7c1f8b90712)

Reply:
https://github.com/PyCQA/pycodestyle/pull/1106/files

Reply:
"ambiguous identifier" rule now applies to lambdas with the pycodestyle bump.